web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -15,7 +15,6 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
     PENDING_EVENT_ATTR: 'PENDING_EVENT',
 
     constructor: function(cfg) {
-
```

Comment:
As a note I wouldn't let anyone apply a 'developer discretion'  code style like this (to a whole file even though they only touched a small part of it) even if the new code style was my preference, too. The point is not to reformat back-and-forth and it's not to apply your style to other people's work. If we've agreed on 'developer discretion' it means we all need to be happy with either.